packages/gradient_boosting_model/gradient_boosting_model/processing/preprocessors.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class SklearnTransformerWrapper(BaseEstimator, TransformerMixin):
+    """Categorical data missing value imputer."""
```

Comment:
The transformer wrapper is a wrapper around sklearn pre processing transformers, so that we can select variables. We need to replace the text in the function:
" Wrapper for Scikit-learn pre-processing transformers,like the SimpleImputer() or OrdinalEncoder(), to allow the use of the transformer on a selected group of variables"

Reply:
updated